src/App.js

```diff
@@ -145,7 +145,7 @@ export class App extends Component {
                             href="https://loja.queplanta.com/"
                             component="a"
                           >
-                            Loja{" "}
+                            Loja&nbsp;
```

Comment:
@ficast qual sentido disso?

Reply:
this. da forma anterior {" "} não reconhece o espaço em branco, tentei outras formas mas a única que funcionou foi esse caractere &nbsp

![image](https://user-images.githubusercontent.com/61601716/100143658-6cb7c900-2e74-11eb-9fb4-b1f8ac24efa1.png)